### PR TITLE
🧪 add unit tests for the image-controller

### PR DESCRIPTION
**Pull request by mevansgh1**

### CODE REVIEW

File: docs-markdown/src/controllers/image-controller.ts [hunk 1]
```diff
@@ -63,10 +63,6 @@ export function pickImageType() {
                 applyImage();
                 commandOption = "image";
                 break;
-            case "image":
```
Comment (mevansgh1) [inline docs-markdown/src/controllers/image-controller.ts:66]: unreachable code, case statement above is the same, git blame shows https://github.com/microsoft/vscode-docs-authoring/commit/576a297808852c8f152ebac307a9fc86c9a497c9

**Comment by mevansgh1** [comment]
fyi, I see my PR is failing the CI build check.  I run this on Mac OS.  I was able to reproduce the tet failure on a Windows VM.  As soon as I have time I will submit an update to the PR to fix the test.

**Comment by mamccrea** [comment]
https://github.com/microsoft/vscode-docs-authoring/issues/482

File: docs-markdown/src/controllers/image-controller.ts (hunk 1 above)
Comment (bharney0) [reply docs-markdown/src/controllers/image-controller.ts:66]: Good call.

**Comment by bharney0** [comment]
@mevansgh1 Thank you for your contribution. These changes look good. Im going to go ahead an merge them. We appreciate the work your doing.

**Pull request merged**